resources/Materials/TestSuite/stdlib/texture/tokenGraph.mtlx

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<materialx version="1.38">
+   <nodegraph name="Tokenized_Image" >
+      <token name="Image_Resolution" type="string" uniform="true" value="2k" uiname="Image Resolution" />
```

Comment:
Why do you specify an Image_Resolution token that is never used?

Reply:
I've updated with a test to show usage of 1 or both tokens.